cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -1639,9 +1642,12 @@ macro(build_rapidjson)
   message(STATUS "Building rapidjson from source")
   set(RAPIDJSON_PREFIX
       "${CMAKE_CURRENT_BINARY_DIR}/rapidjson_ep/src/rapidjson_ep-install")
-  set(RAPIDJSON_CMAKE_ARGS -DRAPIDJSON_BUILD_DOC=OFF -DRAPIDJSON_BUILD_EXAMPLES=OFF
-                           -DRAPIDJSON_BUILD_TESTS=OFF
-                           "-DCMAKE_INSTALL_PREFIX=${RAPIDJSON_PREFIX}")
+  set(RAPIDJSON_CMAKE_ARGS
+      ${EP_COMMON_CMAKE_ARGS}
```

Comment:
I suspect RapidJSON isn't the only ep that wasn't getting EP_COMMON_CMAKE_ARGS passed to it but I didn't do a thorough audit as it was the only one that was causing my issue.

Reply:
Since rapidjson is header-only which of these COMMON_CMAKE_ARGS actually have an impact?

Reply:
`CMAKE_EXPORT_NO_PACKAGE_REGISTRY`, I believe. Before I added the common cmake args, it was writing to `~/.cmake/packages/RapidJSON`, and after, it does not.